frontends/precinct-scanner/src/components/replace_ballot_bag_screen.tsx

```diff
@@ -17,12 +17,14 @@ export function ReplaceBallotBagScreen({
   onComplete,
 }: Props): JSX.Element {
   const [confirmed, setConfirmed] = useState(false);
+  const [onCompleteCalled, setOnCompleteCalled] = useState(false);
 
   useEffect(() => {
-    if (confirmed && !pollWorkerAuthenticated) {
+    if (confirmed && !onCompleteCalled) {
       onComplete();
```

Comment:
note to self: now we call `onComplete` from the `ReplaceBallotBagScreen` as soon as it's confirmed, not waiting until the PW card is removed.